docs/chapters/baseplatform/obtaining-a-PELUX-release.rst

```diff
@@ -0,0 +1,35 @@
+Obtaining a PELUX release
+=========================
+
+The following manifests can be used for a build:
+
+* `pelux-intel.xml` - For building the `core-image-pelux` image for Intel
+* `pelux-intel-qt.xml` - For building the `core-image-pelux-qt` image, which is the baseline with QtAS
+* `pelux-rpi.xml` - For building the `core-image-pelux` image for Raspberry Pi 3
+
+Variables:
+
+* Manifest, refers to what `<manifest-name>.xml` file you want to use, for example `pelux-intel.xml`. Each hardware platform targeted by the PELUX reference has its own manifest describing what other git repositories are needed for the build.
+* Image, refers to what version of PELUX that should be built. Currently there are two versions: `core-image-pelux` and `core-image-pelux-qt`. The latter being a version that includes NeptuneUI and QtApplicationManager.
```

Comment:
Nitpick: "Image, refers to what version of PELUX that should be built." I guess removing "that" would sound better?